trunk/SUAVE/Components/Energy/Networks/Lift_Cruise.py

```diff
@@ -980,6 +988,7 @@ def add_lift_unknowns_and_residuals_to_segment(self, segment, initial_voltage =
         segment.state.conditions.propulsion.lift_rotor_tip_mach          = 0. * ones_row(n_lift_rotors)
         segment.state.conditions.propulsion.lift_rotor_efficiency        = 0. * ones_row(n_lift_rotors)
         segment.state.conditions.propulsion.lift_rotor_motor_efficiency  = 0. * ones_row(n_lift_rotors)
+        segment.state.conditions.propulsion.lift_rotor_y_axis_rotation   = 0. * ones_row(n_lift_rotors)
```

Comment:
update the Data structures here: https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Analyses/Mission/Segments/Conditions/Aerodynamics.py#L132

Reply:
already updated